src/Viewer/README.md

```diff
@@ -35,42 +35,82 @@ const App = () => {
         FILE_VIEW: 1,
     };
 
-    const urlSearchParams = new URLSearchParams(window.location.search, "?");
-
     const [fileInfo, setFileInfo] = useState(null);
     const [theme, setTheme] = useState(THEME_STATES.DARK);
-    const [appMode, setAppMode] = useState();
+    const [appMode, setAppMode] = useState(null);
```

Comment:
Need to update this file with the changes made in App.js, right?

Reply:
Done